tutorials/eda_contents/Computing_Environment/Unix.md

```diff
@@ -0,0 +1,363 @@
+~~~
+
+#### Key Points:
+- "The file system is responsible for managing information on the disk."
+- "Information is stored in files, which are stored in directories (folders)."
+- "Directories can also store other directories, which forms a directory tree."
+- "`cd path` changes the current working directory."
+- "`ls path` prints a listing of a specific file or directory; `ls` on its own lists the current working directory."
+- `pwd` prints the user's current working directory.
+- `whoami` shows the user's current identity.
+- `/` on its own is the root directory of the whole file system.
+- A relative path specifies a location starting from the current location.
+- An absolute path specifies a location from the root of the file system.
+- Directory names in a path are separated with '/' on Unix, but '\\\\' on Windows.
+- '..' means 'the directory above the current one'; '.' on its own means 'the current directory'.
+- Most files' names are `something.extension`. The extension isn't required, and doesn't guarantee anything, 
+but is normally used to indicate the  type of data in the file.
+- Most commands take options (flags) which begin with a '-'.
+
+## Working with Files and Directories
+
+You would have seen basic ways to navigate through files and directories in Unix. Now, let's make them.
+
+To create a directory, we use the command `mkdir` plus the name of the directory you wish to create.
+
+~~~
+/notebook_dir/mypublic$ mkdir example_of_directory
+/notebook_dir/mypublic$ ls
+'Environment'   example_of_directory   Marble.md   Plan.ipynb
+~~~
+
+Note the `mkdir` yielded no output but when we check for the list of files and directories within the current working directory, 
+we see `example_of_directory` was created. 
+Be aware that the `mkdir` command makes the new directory within your current working directory. Ensure you create the directory in 
+the correct location.
+
+## Good names for files and directories
+
+ Complicated names of files and directories can make your life painful
+ when working on the command line. Here we provide a few useful
+ tips for the names of your files.
+
+ 1. Don't use whitespaces.
+
+   Whitespaces can make a name more meaningful but since whitespace is used to break arguments on the command line
+   is better to avoid them on name of files and directories. You can use `-` or `_` instead of whitespace.
+
+ 2. Don't begin the name with `-` (dash).
+
+    Commands treat names starting with `-` as options.
+
+ 3. Stick with letters, numbers, `.` (period), `-` (dash) and `_` (underscore).
+
+    Many other characters have special meanings on the command line. We will learn
+    about some of these during this lesson. There are special characters that can
+    cause your command to not work as expected and can even result in data loss.
+
+ If you need to refer to names of files or directories that have whitespace
+ or another non-alphanumeric character, you should surround the name in quotes (`""`).
```

Comment:
recommend single quotes over double quotes for this since shell scripts may expand certain characters/variables in double quotes

Reply:
updated